packages/edit-site/src/components/sidebar/global-styles-v2/screens/typography-element.js

```diff
@@ -0,0 +1,286 @@
+/**
+ * External dependencies
+ */
+import { isUndefined, startCase } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Button,
+	ButtonGroup,
+	CardBody,
+	Dropdown,
+	DropdownMenu,
+	DropdownMenuItem,
+	DropdownTrigger,
+	FormGroup,
+	Grid,
+	ListGroup,
+	Select,
+	Slider,
+	TextInput,
+	UnitInput,
+	useNavigatorParams,
+	VStack,
+} from '@wordpress/components';
+import {
+	formatCapitalize,
```

Comment:
It's so convenient that we have those in `@wordpress/icons` instead of having to borrow them from `react-icons`.